### ISSUE TITLE
"Failed to resolve: com.github" for androidx package

### ISSUE THREAD

**Issue by raymondctc** [whataa/pandora#50]
It's ok if I use the lib: `implementation 'com.github.whataa:pandora:v2.0.2'`
But for AndroidX, it has an error "Failed to resolve: com.github"

build.gradle:
```
dependencies {
    implementation "com.android.support:multidex:1.0.3"
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    implementation 'androidx.appcompat:appcompat:1.0.2'
    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
    implementation 'com.google.android.material:material:1.1.0-alpha02'
    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'com.android.support.test:runner:1.0.2'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
    implementation 'com.google.firebase:firebase-core:16.0.6'
    implementation 'com.github.whataa:pandora:androidx_v2.0.0'
}
repositories {
    mavenCentral()
    maven { url 'https://jitpack.io' }
}
```

**Comment by whataa** [comment]
Pandora for AndroidX is available from v2.0.2, so you should change it to:
`implementation 'com.github.whataa:pandora:androidx_v2.0.2'`

check [release doc](https://github.com/whataa/pandora/releases) for more detail.

**Comment by whataa** [comment]
stale
